helm_deploy/values-dev.yaml

```diff
@@ -8,6 +8,19 @@ generic-service:
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  allowlist:
```

Comment:
I think adding an allow list without all IPs allowed would limit that to just those ips, let me check